src/pages/Login/Login.js

```diff
@@ -1,7 +1,236 @@
 import React from 'react';
+import styled from 'styled-components/macro';
 
 const Login = () => {
-  return <h1>Login</h1>;
+  return (
+    <LoginWrapper>
+      <Header>
+        <a href="/main">logo</a>
+      </Header>
+      <Container>
+        <Row>
+          <Form>
+            <Inner>
+              <Title>로그인 및 회원가입</Title>
+              <SubTitle>
+                로그인을 통해 파인딩푸드의 다양한 혜택을 누리세요.
+              </SubTitle>
+              <div>
+                <TextID
+                  type="text"
+                  placeholder="휴대전화번호 또는 이메일 입력"
+                ></TextID>
+                <TextPW type="text" placeholder="비밀번호 입력"></TextPW>
```

Comment:
```suggestion
                <TextPW type="password" placeholder="비밀번호 입력" / >
```
type 수정 해 주시고,
태그랑 태그 사이에 컨텐츠가 없으면 self closing 해 주세요~

Reply:
네! type 수정했습니다!